.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,11 @@
-## What kind of a pull request is this?
-
-- QA / Code Review
-<!-- Add other options if appropriate -->
-
+## Description of changes
+<!-- It is often obvious what changed by looking at the code, so it is more helpful to say _why_ it should be changed -->
+<!-- If the Pull Request is not ready to be merged, please use a draft pull request -->
 
 ## Code Checklist
 
-- [ ] The code follows dotkom code style 
-- [ ] The code passes the defined tests
 - [ ] I have added tests for the code I added
 - [ ] I have provided documentation for the code I added
-- [ ] The code is ready to be merged
-
 
 ## (Possible) Breaking Changes
```

Comment:
Rather than a mess of checkboxes I think we could use just a comment that said something like "If you have made any possibly breaking changes, please describe them and what changes should be made to production" (Just phrased better).

Reply:
Basically, it feels like shit to have tons of checkboxes that you don't check. I'd rather remove them if most people shouldn't use them.

Reply:
Agree! Changed it to a new comment now